src/Package/Impl/Utilities/ProjectUtilities.cs

```diff
@@ -42,5 +52,95 @@ public static class ProjectUtilities {
 
             return result;
         }
+
+        public static EnvDTE.Project GetActiveProject() {
```

Comment:
PTVS has a bunch of DTE mocks that might be of use.

https://github.com/Microsoft/PTVS/tree/0a3d9d17008e8d9b8fe2cb40c6c501f13861ccdb/Common/Tests/MockVsTests